config/application.rb

```diff
@@ -6,6 +6,17 @@
 
 require_relative '../lib/version'
 
+def lmo_asset_host
```

Comment:
I have found myself wanting access to this in other parts of the application as well

(namely the emojifier, to ensure that we're displaying images from our server in emails)
I guess with this I could then use `Rails.application.config.action_mailer.asset_host`?